packages/testing/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@firebase/testing",
+  "version": "0.1.0",
```

Comment:
Lets make sure we have the `private` value set to `true` for this module. I don't think we want to publish this quite yet?

Reply:
done